### DISCUSSION TITLE
管理用のWebhookとかほしい

### DISCUSSION THREAD

**Discussion by SanMurakami** [misskey-dev/misskey#7111] [Ideas]
通報とか受けた時に現状だとすぐに気付く手段がないので、WebhookとかでDiscordとかSlackに転送できたら便利な気がする。

通報以外にもユーザー登録とかログイン失敗とかも通知できたら便利そう

**Comment by syuilo** [comment]
関連: https://github.com/syuilo/misskey/issues/6332

**Reply by SanMurakami** [reply]
#6332 はユーザーが利用することを想定してる気がするんだけど、実装した場合の負荷が気になるよね

**Reply by rinsuki** [reply]
負荷的にはユーザーの使うブラウザ数が増える(=webpushの購読が増える)のとそんなに変わらない気がする

**Reply by SanMurakami** [reply]
なるほど
